### ISSUE TITLE
train_opacity.py error

### ISSUE THREAD

**Issue by MultiTrickFox** [noodle-lab/GaussianSpa#6]
Hello I noticed in `train_opacity.py`
at line 130: `loss = optimizingSpa.append_spa_loss(loss, imp_score)`
The `OptimizingSpa` class has no `imp_score`
could you recheck please?

**Comment by zhangseammm** [comment]
Sorry for this. We have corrected it.
